recipes/pyspark/README.md

```diff
@@ -0,0 +1,202 @@
+If all of these are previously uploaded in your org you can directly go to the section `Steps to run training and scoring jobs`
+Make sure that the name of the schema and the dataset ids match the ones being supplied in `Sample config json`
+
+### Upload Schema
+
+Upload the schema of the retail data (input data) from: 
+`https://github.com/adobe/experience-platform-dsw-reference/blob/master/datasets/retail/DSWRetailSales.xdm.json`
+
+
+        curl -X POST \
+             https://platform.adobe.io/data/foundation/catalog/xdms/_customer/DSWRetailSales \
+             -H 'Authorization: Bearer {token} \ - Get the token from cookies of the UI 
+             -H 'Content-Type: application/json' \
+             -H 'cache-control: no-cache' \
+             -H 'x-api-key: acp_machineLearning_customer' \
+             -H 'x-gw-ims-org-id: 20656D0F5B9975B20A495E23@AdobeOrg' \
+             -d '{
+                   "title": "DSWRetailSales",
+                   "description": "string",
+                   "type": "object",
+                   "properties": {
+                     "date": {
+                       "id": "date",
+                       "type": "string"
+                     },
+                     "store": {
+                       "id": "store",
+                       "type": "integer"
+                     },
+                     "storeType": {
+                       "id": "storeType",
+                       "type": "string"
+                     },
+                     "weeklySales": {
+                       "id": "weeklySales",
+                       "type": "number"
+                     },
+                     "storeSize": {
+                       "id": "storeSize",
+                       "type": "integer"
+                     },
+                     "temperature": {
+                       "id": "temperature",
+                       "type": "number"
+                     },
+                     "regionalFuelPrice": {
+                       "id": "regionalFuelPrice",
+                       "type": "number"
+                     },
+                     "markdown": {
+                       "id": "markdown",
+                       "type": "number"
+                     },
+                     "cpi": {
+                       "id": "cpi",
+                       "type": "number"
+                     },
+                     "unemployment": {
+                       "id": "unemployment",
+                       "type": "number"
+                     },
+                     "isHoliday": {
+                       "id": "isHoliday",
+                       "type": "boolean"
+                     }
+                   }
+             }
+### Upload Dataset from UI             
+
+Upload the parquet file from here:
+`https://github.com/adobe/experience-platform-dsw-reference/blob/master/datasets/retail/DSWRetailSales.parquet`
```

Comment:
What version is this based on (XDM 0.9.9.?)  Does it need updating?

Reply:
Will capture this validation as part of the ticket - https://jira.corp.adobe.com/browse/PLATML-4247